### DISCUSSION TITLE
¿How import complete tilemap?, not object-only tilemap (platforms)

### DISCUSSION THREAD

**Discussion by firecrauter** [Kode/Kha#1479] [Q&A] [answered]
I know how to import an object-only tilemap with a modified version of
https://github.com/lewislepton/raccoon/tree/master/Sources/raccoon [(old tutorial)](https://youtu.be/dGjv3U5VS3o?si=REVOn9D5BfJkHmH6)

<img src="https://github.com/Kode/Kha/assets/7044466/f9cb21fb-c532-4536-82a5-1da0c703bede" width=50% height=50%>
<img src="https://github.com/Kode/Kha/assets/7044466/a6a1bb37-28f3-4dbf-8559-096e20a1874c" width=24% height=24%>

Is there a library to import a complete tilemap?, like in this image:
<img src="https://github.com/Kode/Kha/assets/7044466/1fec1d71-7b70-48c0-bf86-5d42db048c76" width=50% height=50%>

**Comment by RobDangerous** [comment, marked as answer]
https://github.com/Kha-Samples/Kha2D includes some code for that and https://github.com/Kha-Samples/TPlayer is a sample that makes use of it.
